### PR TITLE
Redirect on activate page loaded

### LINKED ISSUE #194: Need redirection after user verify their account
Redirect from `/account/activate` to `/` if the verification is `ok`

### PR DESCRIPTION
**Pull request by DylanFlandrin**
## Description

The purpose is to redirect to the login page if activate page is loaded

I added 2 toasters to alert the user of the status of the activation

closes #194

### CODE REVIEW

**Comment by yoannfleurydev** [comment]
closes #194

File: src/spa/account/PageActivate.tsx [hunk 1]
```diff
@@ -16,11 +16,32 @@ export const PageActivate = () => {
...
-
+  const toast = useToast();
+  const navigate = useNavigate();
   useEffect(() => {
     activateAccount({ key: searchParams.get('key') ?? 'KEY_NOT_DEFINED' });
   }, [activateAccount, searchParams]);
 
+  if (isError) {
+    navigate('/');
+    toast({
```
Comment (yoannfleurydev) [inline src/spa/account/PageActivate.tsx:27]: Please use already defined toast to get the same UI and behavior https://github.com/BearStudio/start-ui-web/blob/eb62ec5d0e20d51e17280af690cdfd23a654dae3/src/components/Toast/index.ts#L10

File: src/spa/account/PageActivate.tsx [hunk 2]
```diff
@@ -16,11 +16,32 @@ export const PageActivate = () => {
...
+      title: t('account:activate.feedbacks.activationError.title'),
+      status: 'error',
+      duration: 9000,
+      isClosable: true,
+      position: 'top-right',
+    });
+  }
+  if (isSuccess) {
+    navigate('/');
+    toast({
```
Comment (yoannfleurydev) [inline src/spa/account/PageActivate.tsx:37]: Please use already defined toast function to get the same UI and behavior

https://github.com/BearStudio/start-ui-web/blob/eb62ec5d0e20d51e17280af690cdfd23a654dae3/src/components/Toast/index.ts#L23

**Review by yoannfleurydev — CHANGES_REQUESTED** [review]

**Comment by ImbuPro** [comment]
> Please use already defined toast to get the same UI and behavior https://github.com/BearStudio/start-ui-web/blob/eb62ec5d0e20d51e17280af690cdfd23a654dae3/src/components/Toast/index.ts#L10 

fix

File: src/spa/account/PageActivate.tsx (hunk 1 above)
Comment (DylanFlandrin) [reply src/spa/account/PageActivate.tsx]: fixed

File: src/spa/account/PageActivate.tsx (hunk 2 above)
Comment (DylanFlandrin) [reply src/spa/account/PageActivate.tsx]: fixed

**Review by yoannfleurydev — APPROVED** [review]

**Review by Jessy-BAER — APPROVED** [review]

**Review by yoannfleurydev — CHANGES_REQUESTED** [review]
After re-review this PR, can you please use the onError and onSuccess callback provided by the `mutate` or in this case, the `activateAccount` function?

https://tanstack.com/query/v4/docs/react/reference/useMutation

**Review by yoannfleurydev — APPROVED** [review]

**Pull request merged**